dist/google-music.js

```diff
@@ -279,6 +293,16 @@ proto.rating = {
     if (el && !this.rating._isElSelected(el)) {
       el.click();
     }
+  },
+
+  // Reset the rating
```

Comment:
This comment isn't necessary as it doesn't add any new info (we get the info from the method's name)